### PR TITLE
Add support for prefixes in the trash bin

### PR DESCRIPTION
**Pull request by SkleKng**

### CODE REVIEW

File: EGO Forum Enhancement.user.js [hunk 1]
```diff
@@ -296,11 +392,21 @@ function handleLeadership() {
     generateRedText("80%");
 }
 
+/**
+ * Changes the target of the application links to open in a new tab
+ * @returns void
+ */
+function handleApplicationPage() {
+    document.querySelector('.dataList-row > :nth-child(2)').childNodes[1].setAttribute('target', '_blank');
+    document.querySelector('.dataList-row > :nth-child(2)').childNodes[3].setAttribute('target', '_blank');
```
Comment (blankdvth) [inline EGO Forum Enhancement.user.js:401]: Is it possible to adjust these to use a more lengthy querySelector as opposed to childNodes? The application buttons may be modified by another script, or even updated, which can cause childNodes to point to the wrong element. Same goes for `:nth-child` unless there's no other choice.

File: EGO Forum Enhancement.user.js [hunk 2]
```diff
@@ -288,6 +319,71 @@ function handleBanReport() {
...
+        reason.value = "Discord ID must be verfied in MAUL";
+        explain.value = "In order for you to fix this you'll need to click the MAUL link at the top of the page in the navbar, click \"Edit Game IDs,\" \
+then click the sign in through Discord button under the discord ID section. Once you have done so, please reply to this post!"
+    })
+
+    addPreset("Inappropriate Name", div, function() {
+        reason.value = "Inappropriate Name";
+        explain.value = "As for your name, Please click [URL='https://www.edgegamers.com/account/username']here[/URL] and fill out a name change request. \
+After you fill it out, please wait while your name change request is finalized and the change is completed. \
+Once it is done your application process will resume. If you want to have an understanding on our naming policy inside of eGO please click [URL='https://www.edgegamers.com/threads/378540/']here[/URL].";
```
Comment (blankdvth) [inline EGO Forum Enhancement.user.js:383]: Could we modify these to be more in line with indenting, potentially swapping out `\` for `\n` if needed?

File: EGO Forum Enhancement.user.js [hunk 3]
```diff
@@ -288,6 +319,71 @@ function handleBanReport() {
         addMoveButton(button_group, window.location.href);
 }
 
+/**
+ * Adds "On Hold" templates to the menu and increases the size of the explain box.
+ * @param {HTMLElementEventMap} event 
+ * @returns void
+ */
+function handleOnHold(event) {
+    if(event.target.nodeName != 'DIV' || !event.target.classList.contains('overlay-container'))
```
Comment (blankdvth) [inline EGO Forum Enhancement.user.js:328]: Inconsistent spacing

```suggestion
    if (event.target.nodeName != 'DIV' || !event.target.classList.contains('overlay-container'))
```

File: EGO Forum Enhancement.user.js [hunk 4]
```diff
@@ -288,6 +319,71 @@ function handleBanReport() {
...
+        return;
+
+    // Event may fire twice - add a mark the first time it fires, and ignore the rest
+    var mark = document.createElement('input');
+    mark.type = 'hidden';
+    event.target.append(mark);
+    if(event.target.childNodes.length > 2)
```
Comment (blankdvth) [inline EGO Forum Enhancement.user.js:335]: Inconsistent spacing

```suggestion
    if (event.target.childNodes.length > 2)
```

File: EGO Forum Enhancement.user.js [hunk 5]
```diff
@@ -238,10 +264,15 @@ function handleForumsList() {
                 }
                 var userHref = thread.querySelector('.structItem-cell--main > .structItem-minor > .structItem-parts > li > a');
                 var threadTitle = thread.querySelector('.structItem-cell--main > .structItem-title > a');
+
+                var titleHtml = threadTitle.outerHTML;
+                if(threadTitle.classList.contains('labelLink')) {
```
Comment (blankdvth) [inline EGO Forum Enhancement.user.js:269]: Inconsistent spacing

```suggestion
                if (threadTitle.classList.contains('labelLink')) {
```

File: EGO Forum Enhancement.user.js [hunk 6]
```diff
@@ -318,5 +424,8 @@ function handleLeadership() {
     if (url.match(/^https:\/\/www\.edgegamers\.com\/forums\/?$/))  // Forums List
         handleForumsList();
 
+    if (url.match(/^https:\/\/www\.edgegamers\.com\/application\/\d+\/?$/))  // Application Page
+        handleApplicationPage();
+
```
Comment (blankdvth) [inline EGO Forum Enhancement.user.js:429]: All of the ifs from line 421-427 (one more block above than I've highlighted here) should be swapped for else ifs, those regexes are mutually exclusive.

**Review by blankdvth — CHANGES_REQUESTED** [review]
Generally just some formatting changes here and there. In the `handleForumsList()` function, there's a lot of inconsistent/missing spaces in if statements that need to be fixed, I've highlighted some of them, although the function still contains more.

File: EGO Forum Enhancement.user.js (hunk 2 above)
Comment (SkleKng) [reply EGO Forum Enhancement.user.js:383]: There's no easy way to accomplish this to my knowledge, maybe somebody else can figure this out.

**Review by TheodoreHua — APPROVED** [review]

**Pull request merged**